meshroom/ui/qml/ImageGallery/ImageGallery.qml

```diff
@@ -23,6 +23,7 @@ Panel {
     readonly property string currentItemSource: grid.currentItem ? grid.currentItem.source : ""
     readonly property var currentItemMetadata: grid.currentItem ? grid.currentItem.metadata : undefined
     readonly property int centerViewId: (_reconstruction && _reconstruction.sfmTransform) ? parseInt(_reconstruction.sfmTransform.attribute("transformation").value) : 0
+    readonly property GridView grid: grid
```

Comment:
grid = grid?

Reply:
maybe better to use a different name for each var.
If it's to expose it externally, maybe use "property alias".

Reply:
Done.